api/utils/sshutils/ssh.go

```diff
@@ -250,3 +260,263 @@ func KeysEqual(ak, bk ssh.PublicKey) bool {
 	b := ssh.Marshal(bk)
 	return (len(a) == len(b) && subtle.ConstantTimeCompare(a, b) == 1)
 }
+
+// ConvertToPPK takes a regular RSA-formatted keypair and converts it into the PPK file format used by the PuTTY SSH client.
+// The file format is described here: https://the.earth.li/~sgtatham/putty/0.76/htmldoc/AppendixC.html#ppk
+func ConvertToPPK(priv []byte, pub []byte) ([]byte, error) {
+	// decode the private key from PEM format and extract the exponents
+	privateKeyPemBlock, rest := pem.Decode(priv)
+	if len(rest) > 0 {
+		return nil, trace.Wrap(fmt.Errorf("failed to decode private key, %v bytes left over", len(rest)))
```

Comment:
Could this use `trace.Errorf` instead :)

Reply:
Certainly can! Thanks!